### ISSUE TITLE
The behavior of `useFsWatcher` is confusing

### ISSUE THREAD

**Issue by s3xysteak** [KermanX/reactive-vscode#15]
Now it use `watch` to keep reactive [related code](https://github.com/KermanX/reactive-vscode/blob/main/packages/core/src/composables/useFsWatcher.ts#L69), which made me confused because `watch` do not run effect in the first time.

Hope use `{ immediate: true }` here.

**Comment by s3xysteak** [comment]
A use case:

```ts
const config = defineConfigObject(/* ... */)
useFsWatcher(computed(() => {
  const files = config.xxxPath.map(/* ... */)
  return files.filter(exist)
}))
```

In the use case, user has to update config then the fsWatcher could just work.

**Comment by KermanX** [comment]
Fixed via 3b80aa7a9f8a1437b6a553a47be4a4b7699c410d
